### PR TITLE
Awu issue 19

### PR DESCRIPTION
**Pull request by albertywu**
adds `dataView.onSetItems` event

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "slickgrid",
-  "version": "4.1.1",
+  "version": "4.1.2",
```
Comment (bcherny) [inline bower.json]: you want a minor version bump

File: slick.dataview.js [hunk 2]
```diff
@@ -150,6 +151,7 @@
       updateIdxById();
       ensureIdUniqueness();
       refresh();
+      onSetItems.notify({ "items": items }, null, self);
```
Comment (bcherny) [inline slick.dataview.js]: dont need quotes

**Comment by bcherny** [comment]
:+1:

**Comment by albertywu** [comment]
cool; updated

**Pull request merged**